andela/hc-millennium-falcon
# #156577919 Continuous nagging

#### What does this PR do?
The system nags by the user until he resolve issues they have been notified about

#### Description of Task to be completed?
The system will keep nagging the user every so often until they resolve the job that has gone down.

#### How should this be manually tested?
Run `./manage.py runserver`.
Run `./manage.py sendalerts`.
Create a new check and test the time intervals.

#### Any background context you want to provide?
When a User's job goes down, the user is sent a notification email informing him of the same but he would not be nagged if the check status remained down.

#### What are the relevant pivotal tracker stories?
[156577919](https://www.pivotaltracker.com/n/projects/2162685)
